pydocstringformatter/_utils/issue_template.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+import tokenize
+
+from pydocstringformatter._formatting import Formatter
+from pydocstringformatter._utils.file_diference import compare_formatters
+
+
+def create_gh_issue_template(
+    token: tokenize.TokenInfo, formatters: dict[str, Formatter], filename: str
+) -> str:
+    """Make a template for a GitHub issue.
+
+    Args:
+        token: The token that caused the issue.
+        formatters: The formatters that caused the issue.
+        filename: The filename of the file that caused the issue.
+    """
+    formatter_names = list(formatters)
+    msg = ""
+    if len(formatter_names) > 2:
+        msg = f"""
+Conflicting formatters: {", ".join(formatters)}
+"""
+        diff = None
```

Comment:
Why is `diff` `None` here? Can't we generate a `diff`?

Reply:
I set it to None and it stays that way if there are more than 2 problematic formatters. Otherwise it gets over-written in L49

Reply:
Can't we generate a diff with more than 2 formatters?

Reply:
well ... Im not sure ...
1. If you are talking about making a multi-way diff, I dont believe we can ... I do not know how ...
2. I think we could do several sequential 2-formatter comparissons, but I think the output would be massive
3. since it shows the input token and the problematic formatters I don't think we need to show also a diff when many formatters show problems.

Reply:
Okay, but let's put something there like:
"Diff too intricate to compute for this amount of formatters".